packages/babel-runtime/babel-runtime.js

```diff
@@ -0,0 +1,102 @@
+// The name `babelHelpers` is hard-coded in Babel.  Otherwise we would make it
+// something capitalized and more descriptive, like `BabelRuntime`.
+babelHelpers = {
```

Comment:
@dgreensp I copied this file from https://github.com/meteor/react-packages/blob/master/packages/babel-runtime/babel-runtime.js recently, but if you've made additional changes they may need to be ported here